packages/deployment/app/config.yml

```diff
@@ -8,6 +8,14 @@ default:
   message_adapter_repo: cumulus-message-adapter
   message_adapter_filename: cumulus-message-adapter.zip
 
+  nested_templates:
+    CumulusApiDefault:
+      cfFile: node_modules/@cumulus/deployment/app/cumulus_api.template.yml
+      configFile: node_modules/@cumulus/deployment/app/cumulus_api_default.config.yml
+    CumulusApiV1:
+      cfFile: node_modules/@cumulus/deployment/app/cumulus_api.template.yml
```

Comment:
Should this file be the same as `nested_templates.CumulusApiDefault.cfFile`?

Reply:
yeah, because they share the same template.

Reply:
Basically default and version 1.0 are identical except that they have different api web addresses. V1 has `/v1` in front of all addresses.